GliaWidgetsTests/Sources/OnHoldOverlayVisualEffectView/OnHoldOverlayVisualEffectViewTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+@testable import GliaWidgets
+
+final class OnHoldOverlayVisualEffectViewTests: XCTestCase {
+
+    func test_overlayViewDrawIsExecutedOnMainQueue() {
+        enum Call { case mainQueueAsync, globalQueueAsync }
+        var calls: [Call] = []
+        var gcd = GCD.failing
+        gcd.mainQueue.async = { _ in
+            calls.append(.mainQueueAsync)
+        }
+        gcd.globalQueue.async = { _ in
```

Comment:
Why `gcd.globalQueue.async` also is overwritten if it is not used?

Reply:
Agree that globalQueue check is redundant. I added this check to be sure that it's not called.

Reply:
Fixed 3c62e8478011e44cdeec2e7fa053456067aeef94